cmd/agent/install_script.sh

```diff
@@ -6,7 +6,7 @@
 # using the package manager and Datadog repositories.
 
 set -e
-install_script_version=1.4.0
+install_script_version=1.5.0
```

Comment:
Let's keep this as `1.4.0` and do this in a future PR when we update the changelog
```suggestion
install_script_version=1.4.0
```

Reply:
I'll revert this part last after testing